### PR TITLE
Title changes and other minor fixes

### PR DESCRIPTION
**Pull request by divyenpatel**
- Documentation Site title is changed from `Kubernetes vSphere Cloud Provider` to `vSphere Storage for Kubernetes`.
- Updated `overview.md` and mentioned 

> vSphere Storage for Kubernetes offers a Cloud Provider for Kubernetes which allow Pods to use enterprise grade persistent storage.

- Removed unwanted jekyll files.
- and other minor fixes.

### CODE REVIEW

File: documentation/overview.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Cloud provider provides following interfaces to effectively integrate cloud plat
...
+vSphere Storage for Kubernetes offers a Cloud Provider for Kubernetes which allow Pods to use enterprise grade persistent storage. vSphere has a proven Software Defined Storage(SDS) platform that integrates with block, file and HyperConverged offerings such as VMware VSAN. These storage offerings can exposed as VMFS, NFS, VVol and VSAN datastores. SDS platform in vSphere has enterprise grade features like Storage Policy Based Management(SPBM) which enables customers to guarantee QoS requested by their business critical applications and enforce SLAs. vSphere provides various data services out of the box such high availability and data reliability for containers using Kubernetes.
```
Comment (shahzebnp) [inline documentation/overview.md]: which allow -> which allows

File: documentation/overview.md [hunk 2]
```diff
@@ -32,14 +32,15 @@ Datastores is an abstraction which hides storage details and provide uniform int
...
 ## Kubernetes Storage support
-vSphere Cloud Provider supports every storage primitive exposed by Kubernetes -
-* Volumes
-* Persistent Volumes (PV)
-* Persistent Volumes Claims (PVC)
-* Storage Class
-* Stateful Sets
-
-Persistent volumes requested by stateful containerized applications can be provisioned on VSAN, VVol, VMFS or NFS datastores.
+vSphere Cloud Provider (VCP) supports every storage primitive exposed by Kubernetes -
```
Comment (shahzebnp) [inline documentation/overview.md]: Shouldn't we use vSphere storage for Kubernetes here?

File: documentation/persistent-vols-claims.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 title: Persistent Volumes & Persistent Volumes Claims
 ---
 
-In case of Kubernetes Volumes we know that once the Pod is deleted the specification of the volume in the Pod is also lost. Even though VMDK file persists but from Kubernetes perspective the volume is deleted.
+In case of Kubernetes Volumes, we know that once the Pod is deleted the specification of the volume in the Pod is also lost. Even though VMDK file persists but from Kubernetes perspective the volume is deleted.
```
Comment (shahzebnp) [inline documentation/persistent-vols-claims.md]: can get rid of "we know".  `, once the Pod is deleted...`

Comment (shahzebnp) [inline documentation/persistent-vols-claims.md]: `from the perspective of Kubernetes` or `Kubernetes's perspective`

File: documentation/persistent-vols-claims.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
...
  
-Persistent Volumes API resource solves this problem where PVs have lifecycle independent of the Pods and not created when Pod is run. PVs are unit of storage which we provision in advance, they are Kubernetes objects backed by some storage, vSphere in this case. PVs are created, deleted using kubectl commands.
+Persistent Volumes API resource solves this problem where PVs have lifecycle independent of the Pods and not created when Pod is spawned. PVs are units of storage provisioned in advance, they are Kubernetes objects backed by vSphere storage. PVs are created, deleted using kubectl commands.
```
Comment (shahzebnp) [inline documentation/persistent-vols-claims.md]: `vSphere storage` -> we need to the standard term here, vSphere storage for kubernetes. But Kubernetes word is redundant. Not sure if we can use any abbreviation here for readability.

**Review by shahzebnp — COMMENTED** [review]
Minor changes. Looks good.

File: documentation/persistent-vols-claims.md (hunk 4 above)
Comment (BaluDontu) [reply documentation/persistent-vols-claims.md]: +1

File: documentation/overview.md (hunk 1 above)
Comment (BaluDontu) [reply documentation/overview.md]: HyperConverged -> hyperConverged

File: documentation/overview.md (hunk 2 above)
Comment (BaluDontu) [reply documentation/overview.md]: +1

**Review by BaluDontu — CHANGES_REQUESTED** [review]
Some minor comments.

File: documentation/overview.md (hunk 2 above)
Comment (divyenpatel) [reply documentation/overview.md]: No, Above in the documentation we have mentioned
> vSphere Storage for Kubernetes offers a Cloud Provider (vSphere Cloud Provider - VCP) for Kubernetes which allows Pods to use enterprise grade persistent storage.

This sets the context for VCP. Under `vSphere Storage for Kubernetes` we will have multiple offerings. VCP is just one of them.

File: documentation/persistent-vols-claims.md (hunk 4 above)
Comment (divyenpatel) [reply documentation/persistent-vols-claims.md]: Here `vSphere storage` mean,  actual vsphere datastores (vSan, VVol, vmfs, nfs), and not the `vSphere Storage for Kubernetes` Products.

File: documentation/persistent-vols-claims.md (hunk 3 above)
Comment (divyenpatel) [reply documentation/persistent-vols-claims.md]: Fixed.

Comment (divyenpatel) [reply documentation/persistent-vols-claims.md]: Fixed.

File: documentation/overview.md (hunk 1 above)
Comment (divyenpatel) [reply documentation/overview.md]: Fixed.

File: documentation/existing.md [hunk 5]
```diff
@@ -54,7 +54,7 @@ Note: If Kubernetes Node VMs are created from template VM then `disk.EnableUUID=
 
 #### Step 4: Create Roles, add Privileges to Roles and assign them to the vSphere Cloud Provider user and vSphere entities
 
-Note: if you want to use Administrator account then this step can be skipped.
+Note: if vCenter Administrator account is specified in the `vsphere.conf` file, then this step can be skipped.
```
Comment (BaluDontu) [inline documentation/existing.md]: if -> If

File: documentation/existing.md [hunk 6]
```diff
@@ -96,9 +96,11 @@ vSphere Cloud Provider interacts with vCenter through the user configured in vSp
...
 
-In some cases, however, you may still want to customize the privileges for vSphere Cloud Provider user:
-1. SPBM support in vSphere Cloud Provider was introduced in Kubernetes 1.7 release. If you are still using 1.6 or older release, you may want to remove SPBM related privileges to keep required privileges to a minimal.
-2. Kubernetes RBAC mode was stabilized as of 1.8 release. If you are using 1.7 or older release, you may still want to customize the privileges for vSphere Cloud Provider user based on different use cases. It's recommended to assign different roles to the same user instead of switching to a different user, in which case you'll need to restart kubelet, which might be a concern in production environment.
+In some cases, however, Administrator may still want to customize the privileges for vSphere Cloud Provider user:
```
Comment (BaluDontu) [inline documentation/existing.md]: You can remove however.

File: documentation/faqs.md [hunk 7]
```diff
@@ -6,62 +6,69 @@ title: FAQs
...
+## Can Reclaim Policy be set to `Retain` on dynamic PVs once volumes are provisioned with default reclaim policy `delete`?
+If the volume was dynamically provisioned, then the default reclaim policy is set to “delete”. This means by default, when the PVC is deleted, the underlying PV and storage asset will also be deleted.
+If volume needs to be retained, then reclaim policy needs to be changed from “delete” to “retain”. Please refer this [link](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) for more details.
+
+
+## How to resize dynamic volumes once provisioned?
 Support for resizing existing dynamic volume is not yet there.
-Proposal is out for [review.](https://github.com/gnufied/community/blob/91b41028182a5291b4eccbf88f8065f66b2b7eed/contributors/design-proposals/grow-volume-size.md)
+Proposal is avaiallbe [here](https://github.com/gnufied/community/blob/91b41028182a5291b4eccbf88f8065f66b2b7eed/contributors/design-proposals/grow-volume-size.md). vSphere Cloud Provider does not support resize of volume. Please track this [issue](https://github.com/vmware/kubernetes/issues/168).
```
Comment (BaluDontu) [inline documentation/faqs.md]: avaiallbe -> available

File: documentation/guestbook.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 title: Running Stateful application - Guestbook App
 --- 
  
-Guestbook is PHP application with Redis as backend. In this section, we will demonstrate how to use  Kubernetes deployed on vSphere  to run Guestbook Application with persistent storage. At the end of this demo, you will have a sample guestbook app running inside Kubernetes where the data is resident inside VMDKs managed by vSphere.
+Guestbook is PHP application with Redis as backend. In this section, we will demonstrate how to use  Kubernetes deployed on vSphere to run Guestbook Application with persistent storage. At the end of this demo, a sample guestbook app will be available inside Kubernetes where the data is resident inside VMDKs managed by vSphere.
 
 The data in the VMDK is independent of the lifecycle of the pods and persists even if pods are deleted.
```
Comment (BaluDontu) [inline documentation/guestbook.md]: remove this.

File: documentation/mongodb.md [hunk 9]
```diff
@@ -19,7 +19,7 @@ parameters:
  diskformat: thin
 ```
```
Comment (BaluDontu) [inline documentation/mongodb.md]: Remove this and other places.

File: documentation/faqs.md [hunk 10]
```diff
@@ -6,62 +6,69 @@ title: FAQs
...
 
+## How can disk.EnableUUID enabled on Node VMs? 
+To enable disk.EnableUUID option for the Node VM, Select VM and select `Edit Settings`. Find
+Customize hardware -> VM Options -> Configuration Parameters -> Edit Configuration. Check to see if the parameter disk.EnableUUID is set, if it is there then make sure it is set to TRUE. If the parameter is not there, select Add Row and add it. This can be set on the template VM before cloning node VMs from the template.
 
-## Can we have the disk uuid enabled by default, so we don't need to do it machine by machine? What would be the risks of having it by default.
-You can enable disk UUID by default, while creating VM. Just add this parameter at
-Customize hardware -> VM Options -> Configuration Parameters -> Edit Configuration
 
 ## Is it mandatory to have all the machines in the same directory?
```
Comment (BaluDontu) [inline documentation/faqs.md]: for 1.9 its not a requirement.
for 1.8 and below, it is a requirement.

File: documentation/faqs.md [hunk 11]
```diff
@@ -6,62 +6,69 @@ title: FAQs
...
 Current code base requires Master and Node VMs to be present under one VM folder. Each Kubernetes Cluster deployed in vSphere, should be placed in their respective VM folder, else under root folder.
```
Comment (BaluDontu) [inline documentation/faqs.md]: Can you rename the question from Can Kubernetes Cluster span across multiple vCenters? to 
Can vSphere cloud provider Kubernetes Cluster spanning across multiple vCenters?

Also add question for
Can vSphere cloud provider Kubernetes Cluster spanning across multiple Datacenters?

**Review by BaluDontu — CHANGES_REQUESTED** [review]
Some more comments.

**Review by shahzebnp — APPROVED** [review]
LGTM

File: documentation/faqs.md (hunk 11 above)
Comment (divyenpatel) [reply documentation/faqs.md]: Fixed.

File: documentation/guestbook.md (hunk 8 above)
Comment (divyenpatel) [reply documentation/guestbook.md]: Removed  spaces.

File: documentation/mongodb.md (hunk 9 above)
Comment (divyenpatel) [reply documentation/mongodb.md]: fixed.

**Comment by divyenpatel** [comment]
@BaluDontu I have addressed all your review comments. Please take a look again.

**Review by BaluDontu — APPROVED** [review]
LGTM

**Pull request merged**